content/en/tracing/guide/injectors.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How the Injector Works with Single Step Instrumentation
+further_reading:
+- link: tracing/trace_collection/automatic_instrumentation/single-step-apm
+  tag: Documentation
+  text: Single Step APM Instrumentation
+---
+
+## Overview
+
+The injector is a shared library that automatically instruments applications at runtime. With Single Step Instrumentation, the injector automates environment setup and tracer initialization, eliminating the need for manual configuration.
```

Comment:
add link to https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/?tab=linuxhostorvm